zebra/rt_netlink.c

```diff
@@ -406,6 +406,29 @@ vrf_id_t vrf_lookup_by_table(uint32_t table_id, ns_id_t ns_id)
 	return VRF_DEFAULT;
 }
 
+static struct zebra_vrf *vrf_lookup_by_table_id(uint32_t table_id)
```

Comment:
don't really care for the copy-paste of the existing function logic. can't we have one version of the logic that knows the rules for iterating through the vrfs, and then use that to return the different pieces of info? and really, that should be in the zebra_vrf module, shouldn't it?